packages/bruno-electron/src/ipc/network/index.js

```diff
@@ -514,6 +569,19 @@ const registerNetworkIpc = (mainWindow) => {
       );
       const response = await axiosInstance(preparedRequest);
 
+      await runPostResponse(
```

Comment:
On first thought I am not sure if running post request stuff makes sense.
At least my expectation is that post request scripts are written to run on the actual response of the request (i.e. extract data from the response, run assertions on the response). If now suddenly the response is the introspection data, I'd expect that those scripts and assertions will definitively fail or result in unexpected behaviour.